### PR TITLE
[QA] regen with DPG

### PR DESCRIPTION
**Pull request by kristapratico**
#25300 #25301

### CODE REVIEW

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md [hunk 1]
```diff
@@ -20,25 +20,58 @@ autorest
...
+
+
+## Runtime
+
+These settings apply only when `--tag=release_runtime_1_1_preview` is specified on the command line.
+
+```yaml $(tag) == 'release_runtime_1_1_preview'
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/34a2c0723155d134311419fd997925ce96b85bec/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering.json
+output-folder: ../azure/ai/language/questionanswering
+models-mode: msrest
```
Comment (iscai-msft) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:53]: @kristapratico I believe we want to eventually add new DPG models for authoring. Are we able to move the existing QnA models over to DPG as well / are there plans to?

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md [hunk 2]
```diff
@@ -111,4 +144,133 @@ directive:
...
+  - from: swagger-document
+    where: $["paths"]["/query-knowledgebases/projects/{projectName}"]["patch"]
+    transform: >
+        $["operationId"] = "createProject";
+  - from: swagger-document
+    where: $["paths"]["/query-knowledgebases/projects/{projectName}"]["delete"]
+    transform: >
+        $["operationId"] = "deleteProject";
+  - from: swagger-document
+    where: $["paths"]["/query-knowledgebases/projects/{projectName}/:export"]["post"]
```
Comment (iscai-msft) [inline sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:173]: you've already done the directive, but in the future you can use built in directives to save yourself some code and to make the readme easier to read https://github.com/Azure/autorest/blob/main/docs/generate/built-in-directives.md#rename-operation

**Review by iscai-msft — APPROVED** [review]

File: sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md (hunk 1 above)
Comment (kristapratico) [reply sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:53]: I have not heard plans. This release is intended to stabilize the authoring client. Are DPG models soon to be GA?

Comment (kristapratico) [reply sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md:53]: Discussed offline. Authoring doesn't use models yet (so adding DPG models in the future is not breaking). Runtime already GA'd with msrest models.

**Pull request merged**